blocks/explore-articles/explore-articles.js

```diff
@@ -153,79 +135,72 @@ const buildFirstArticles = (art, section) => {
 };
 
 const buildArticleList = (articles) => {
-  const groupedArticles = divideArray(articles, articlesPerChunk);
+  const groupedArticles = divideArray(articles, artsPerChunk);
   const articleGroups = getArticleGroups(groupedArticles);
-  const totalArticlesNumber = addAllArrays(groupedArticles);
+  const totalArts = addAllArrays(groupedArticles);
   const amountOfGroups = articleGroups.length;
 
-  const paginationSection = createElement('div', { classes: 'pagination-section' });
   const articlesSection = createElement('div', { classes: `${blockName}-articles` });
-
-  const amountOfArticles = createElement('p', { classes: 'article-amount' });
-  amountOfArticles.textContent = (totalArticlesNumber !== 0) ? `${totalArticlesNumber} articles` : getTextLabel('No article Message');
-
-  paginationSection.append(amountOfArticles);
-  articlesSection.append(paginationSection);
-
-  const moreSection = createElement('div', { classes: `${blockName}-more` });
-  const moreButton = createElement('button', { classes: 'more-btn' });
-  moreButton.textContent = getTextLabel('Load more articles button');
-  moreButton.addEventListener('click', (evt) => loadMoreArticles(evt, articleGroups, amountOfGroups));
-  if (totalArticlesNumber > articlesPerChunk) moreSection.append(moreButton);
+  const articlesContent = document.createRange().createContextualFragment(`
+      <div class="pagination-section">
+        <p class="article-amount">${totalArts !== 0 ? `${totalArts} articles` : getTextLabel('No article Message')}</p>
```

Comment:
if this line is too long with the previous variable name, put the P inner content in a new line between the 2 p tags